### PR TITLE
Fix/dqa

### PR DESCRIPTION
**Pull request by Camwyn**
[TEC-3737]

Addresses issues in [EXT-241]

- Alter "get tickets" to indicate sold-out events. Switch to "RSVP now" for RSVPs
- Fix ticket link alignment.
- Fix duped multiday events.
- Fix date alignment. 
- Fix ugly permalinks on activation.
- Fix styling with vertical filterbar.
- Fix Timezones!

https://d.pr/i/Cy1XEg

[EXT-241]: https://moderntribe.atlassian.net/browse/EXT-241
[TEC-3737]: https://moderntribe.atlassian.net/browse/TEC-3737

### CODE REVIEW

File: src/Tribe/Views/Summary_View.php [hunk 1]
```diff
@@ -122,20 +123,50 @@ protected function setup_template_vars() {
...
-		$formatted_start_date_beginning = tribe_beginning_of_day( $event->dates->start->format( Dates::DBDATEFORMAT ), tribe_get_option( 'dateWithoutYearFormat' ) );
-		$formatted_end_date_ending      = tribe_beginning_of_day( $event->dates->end->format( Dates::DBDATEFORMAT ), tribe_get_option( 'dateWithoutYearFormat' ) );
+		$format = tribe_get_option( 'dateWithoutYearFormat' );
+		$formatted_start_date_beginning = tribe_beginning_of_day( $event->dates->start->format( Dates::DBDATEFORMAT ), $format );
+		$formatted_end_date_ending      = tribe_beginning_of_day( $event->dates->end->format( Dates::DBDATEFORMAT ), $format );
+		$formatted_group_date           = tribe_beginning_of_day( $group_date, $format );
+
+		$is_multiday_start = false !== $event->multiday && $formatted_group_date === $formatted_start_date_beginning;
+		$is_multiday_end   = false !== $event->multiday && $formatted_group_date === $formatted_end_date_ending;
+		$is_all_day        = tribe_event_is_all_day( $event );
```
Comment (bordoni) [inline src/Tribe/Views/Summary_View.php]: `$event->all_day`

**Review by bordoni — CHANGES_REQUESTED** [review]

File: src/Tribe/Views/Summary_View.php (hunk 1 above)
Comment (Camwyn) [inline src/Tribe/Views/Summary_View.php]:
```suggestion
		$is_all_day        = $event->all_day;
```

**Review by bordoni — APPROVED** [review]

File: plugin.php [hunk 2]
```diff
@@ -90,4 +90,10 @@ function tribe_extension_summary_view_activation() {
 	$enabled_views[] = 'summary';
 
 	tribe_update_option( 'tribeEnableViews', $enabled_views );
+
+	// Fixes "ugly permalinks" on activation.
+	// Check seems silly, but might as well not do it if it's not needed.
+	if ( get_option('permalink_structure') ) {
+		flush_rewrite_rules();
```
Comment (bordoni) [inline plugin.php]: `set_transient( '_tribe_events_delayed_flush_rewrite_rules', 'yes', 0 );`
Which will trigger a flush at the correct moment.

File: src/Tribe/Views/Summary_View.php [hunk 3]
```diff
@@ -122,20 +86,50 @@ protected function setup_template_vars() {
...
 	}
 
-	protected function add_view_specific_properties_to_event( $event ) {
+	protected function add_view_specific_properties_to_event( $event, $group_date ) {
 		$start_date = tribe_beginning_of_day( $event->dates->start->format( Dates::DBDATEFORMAT ), Dates::DBDATEFORMAT );
 		$end_date   = tribe_beginning_of_day( $event->dates->end->format( Dates::DBDATEFORMAT ), Dates::DBDATEFORMAT );
 
-		$formatted_start_date_beginning = tribe_beginning_of_day( $event->dates->start->format( Dates::DBDATEFORMAT ), tribe_get_option( 'dateWithoutYearFormat' ) );
-		$formatted_end_date_ending      = tribe_beginning_of_day( $event->dates->end->format( Dates::DBDATEFORMAT ), tribe_get_option( 'dateWithoutYearFormat' ) );
+		$format = tribe_get_option( 'dateWithoutYearFormat' );
```
Comment (bordoni) [inline src/Tribe/Views/Summary_View.php]: `tribe_get_date_option( 'dateWithoutYearFormat', Tribe__Date_Utils::DBDATEFORMAT )`

File: src/Tribe/Views/Summary_View.php [hunk 4]
```diff
@@ -122,20 +86,50 @@ protected function setup_template_vars() {
...
+		// Make "middle" days of a multi-day event all-day events.
+		if (
+			false !== $event->multiday
+			&& ! $is_multiday_start
+			&& ! $is_multiday_end
+		) {
+			$is_all_day = true;
+		}
+
+		$end_date = $event->dates->end_display->format( get_option( 'time_format' ) );
```
Comment (bordoni) [inline src/Tribe/Views/Summary_View.php]: Save `get_option( 'time_format' )` to a variable and re-use it.

**Review by bordoni — CHANGES_REQUESTED** [review]

File: plugin.php (hunk 2 above)
Comment (Camwyn) [reply plugin.php]: Thanks, this was giving me a hard time!

**Review by bordoni — APPROVED** [review]

**Pull request merged**